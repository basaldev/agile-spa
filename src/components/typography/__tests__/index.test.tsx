import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Typography } from '../index';

describe('Typography', () => {
  test('render Typography', () => {
      const wrapper = shallow(
        <Typography content="content" />
      );
      expect(toJson(wrapper)).toMatchSnapshot();
  });
});
