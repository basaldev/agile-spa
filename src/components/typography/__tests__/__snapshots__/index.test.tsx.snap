// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Typography render Typography 1`] = `
<p>
  content
</p>
`;
